### PR TITLE
Add sigma reference

### PR DESCRIPTION
**Pull request by frack113**
Just add missing SigmaHQ references

### CODE REVIEW

File: yml/OSBinaries/Ttdinject.yml [hunk 1]
```diff
@@ -24,6 +24,7 @@ Full_Path:
 Code_Sample:
   - Code:
 Detection:
+  - Sigma: https://github.com/SigmaHQ/sigma/blob/35a7244c62820fbc5a832e50b1e224ac3a1935da/rules/windows/image_load/image_load_tttracer_mod_load.yml
```
Comment (wietze) [inline yml/OSBinaries/Ttdinject.yml:27]: Did you perhaps mean to add this one to the `tttracer` entry? All the resources in the Sigma rule point to that executable.

**Review by wietze — CHANGES_REQUESTED** [review]
Thanks for this, one question about one of the rules (see below)

File: yml/OSBinaries/Ttdinject.yml (hunk 1 above)
Comment (frack113) [reply yml/OSBinaries/Ttdinject.yml:27]: This sigma rule detect load dll '\ttdrecord.dll' or '\ttdwriter.dll' or  '\ttdloader.dll'.
Ttdinject.exe load TTDLoader.dll so the rule trigger.

Comment (frack113) [reply yml/OSBinaries/Ttdinject.yml:27]: As it can be disturbing, I will work on a rule with Ttdinject.exe as parent this week .

Comment (frack113) [reply yml/OSBinaries/Ttdinject.yml]: It is fix with the new rules 😄

Comment (wietze) [reply yml/OSBinaries/Ttdinject.yml]: Great stuff, looking good 👌

**Comment by wietze** [comment]
Great additions, thank you for your contribution

**Pull request merged**